collect_app/src/main/java/org/odk/collect/android/geo/OsmDroidMapFragment.java

```diff
@@ -381,10 +381,11 @@ public void removeFeature(int featureId) {
 
     @Override
     public void clearFeatures() {
-        map.getOverlays().clear();
-        addAttributionAndMapEventsOverlays();
-        map.getOverlays().add(myLocationOverlay);
-        map.invalidate();
+        if (map != null) {  // during Robolectric tests, map will be null
```

Comment:
Do we need this check here? We don't have any Robolectric tests for this class as far as I'm aware.